test/unit/metadata.js

```diff
@@ -453,6 +453,26 @@ describe('Image metadata', function () {
       });
   });
 
+  it('Applies custom output ICC profile', function (done) {
+    sharp(fixtures.inputJpg)
+      .withMetadata({ profile: fixtures.path('hilutite.icm') })
+      .toBuffer(function (err, buffer) {
+        if (err) throw err;
+        sharp(buffer).metadata(function (err, metadata) {
+          if (err) throw err;
+          assert.strictEqual(true, metadata.hasProfile);
+          // ICC
+          assert.strictEqual('object', typeof metadata.icc);
+          assert.strictEqual(true, metadata.icc instanceof Buffer);
+          const profile = icc.parse(metadata.icc);
+          assert.strictEqual('object', typeof profile);
+          assert.strictEqual('RGB', profile.colorSpace);
+          assert.strictEqual('Perceptual', profile.intent);
+        });
+        fixtures.assertSimilar(fixtures.path('expected/hilutite.jpg'), fixtures.inputJpg, { threshold: 1 }, done);
```

Comment:
This appears to be asserting that two test fixtures are equivalent - is this intentional? Given this feature relates to colour, the `fixtures.assertMaxColourDistance` helper with the output of the operation might be more appropriate here.

Reply:
I didn't spot the colour distance helper - that makes a lot more sense for this scenario. The `expected/hilutite.jpg` is one I generated by applying `icc_transform()` with the `hilutite.icm` profile - I realise my mistake looking at it now, should be comparing the output 🤦